roles/reproducer/tasks/configure_crc.yml

```diff
@@ -5,6 +5,66 @@
   ansible.builtin.slurp:
     path: /etc/ci/env/networking-environment-definition.yml
 
+- name: Configure CRC services
+  delegate_to: crc-0
+  vars:
+    _parsed: "{{ _net_conf.content | b64decode | from_yaml}}"
+    _crc: "{{ _parsed.instances['crc-0'].networks.ctlplane }}"
+  block:
+    - name: Check which dnsmasq config we must edit
+      register: _dnsmasq
+      ansible.builtin.stat:
+        path: '/srv/dnsmasq.conf'
+        get_attributes: false
+        get_checksum: false
+        get_mime: false
+
+    - name: Configure local DNS for CRC pod
+      become: true
+      vars:
+        _config_file: >-
+          {{
+            _dnsmasq.stat.exists |
+            ternary(_dnsmasq.stat.path, '/etc/dnsmasq.d/crc-dnsmasq.conf')
+          }}
+      block:
+        - name: Configure local DNS for CRC pod
+          register: last_modification
+          ansible.builtin.replace:
+            path: "{{ _config_file }}"
+            regexp: "192.168.130.11"
+            replace: "{{ _crc.ip_v4 }}"
+
+        - name: Configure local DNS for CRC pod with ip_v6
+          when:
+            - _crc.ip_v6 is defined
+            - _crc.ip_v6 | length > 0
+          ansible.builtin.lineinfile:
+            path: "{{ _config_file }}"
+            line: "address=/{{ item  }}/ {{ _crc.ip_v6 }}"
+          loop:
+            - apps-crc.testing
+            - api.crc.testing
+            - api-int.crc.testing
+
+
+        - name: Ensure dnsmasq listens on correct interfaces
+          vars:
+            _listen_adresses: >-
+              {%- if _crc.ip_v6 is defined and _crc.ip_v6 | length > 0 -%}
+              _crc.ip_v6
```

Comment:
you're not building a list I think here, but a string with a space between both elements. So no `join(',')` needed

Reply:
yes, I kept the comma because i thought the `listen-address` field in the dnsmasq config requires a comma-separeted string

Reply:
indeed, that did not do what i expected, thanks @cjeanner I'll update